docs/source/examples/de_sscan.ipynb

```diff
@@ -0,0 +1,1505 @@
+   "execution_count": 23,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0\n"
+     ]
+    }
+   ],
+   "source": [
+    "from ophyd import EpicsSignal\n",
+    "\n",
+    "scan_button = EpicsSignal(f\"{IOC}scan1.EXSC\", name=\"scan_button\")\n",
+    "scan_button.wait_for_connection()\n",
+    "print(f\"{scan_button.get()}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.A. wait with a polling loop\n",
+    "\n",
+    "Write a bluesky plan that pushes the button, waits a brief moment for the scan\n",
+    "to start, then waits for the scan to end.  This simple example waits by\n",
+    "periodically polling the value of the button.\n",
+    "\n",
+    "Here, `scan_button.get(use_monitor=False)` forces ophyd to get a new value\n",
+    "from EPICS (and not to rely on the monitor value being updated promptly).\n",
+    "Probably not necessary to use `use_monitor=False`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def run_sscan():\n",
+    "    yield from bps.mv(scan_button, 1)\n",
+    "    print(\"Started scan ...\")\n",
+    "    yield from bps.sleep(0.1)  # wait for scan to start\n",
+    "    while scan_button.get(use_monitor=False) != 0:\n",
+    "        yield from bps.sleep(0.1)\n",
+    "    print(\"Scan done.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Execute the `run_sscan()` plan using the bluesky RunEngine `RE`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Started scan ...\n",
+      "Scan done.\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "()"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "RE(run_sscan())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To get the data from the `sscan` record, we use `ophyd.EpicsSignalRO` to connect with the arrays.  We'll do that in a later section of this notebook.\n",
+    "\n",
+    "### 1.B. wait with a Status object\n",
+    "\n",
+    "Let's eliminate that polling loop.  This will make the plan respond more quickly\n",
+    "when the button changes value.  The `ophyd.status.Status` class (runs in the\n",
+    "background) reports when the scan has finished or times out.  We define an inner\n",
+    "`watch_the_button()` function to detect when the button goes from `1` to `0`\n",
+    "(the scan stops).  We setup a subscription (*after* we start the move) to\n",
+    "monitor the button value and respond immediately.  \n",
+    "\n",
+    "We wait for the status object to finish with `st.wait()`.  In a bluesky plan,\n",
```

Comment:
```suggestion
    "We wait for the status object to finish with `st.wait()` where `wait(timeout=None)` blocks until the action completes; when the action has finished successfully, returns `None`; if the action has failed, raises an exception.
    
      In a bluesky plan,\n",
```

Reply:
I'll add this locally to avoid a potential merge conflict with other edits on this text block.

Reply:
I got this from the documentation of Status. Now re-reading I wonder _blocks what_?